src/evaluator/runtime.tsx

```diff
@@ -79,6 +79,7 @@ export async function exec({
       throw error
     }
   }
+  writeStuckFlag(true)
```

Comment:
🤔

Reply:
This was here, dunno why, but decided not to remove just in case :)

Reply:
It is allow to handle stuck code. For example infinite while loop. When you restart page, there will be `throw new Error("you're stuck")` or smth like